migration_steps/shared/db_helpers.py

```diff
@@ -125,42 +152,51 @@ def copy_schema(
         )
 
     log.debug("Modify")
-    with fileinput.FileInput(str(schema_dump), inplace=True) as file:
-        for line in file:
-            print(line.replace(from_schema, to_schema), end="")
```

Comment:
since you've got rid of this is is it necessary to rename and unrename the schema?

Reply:
(pretend I commented on the actual offending line)

Reply:
Oh, I think I know what you mean - technically not necessary because we're not seeing any errors related to the schema names. I just wanted to get rid of `line.replace(from_schema, to_schema)` which in my opinion is just as bad as
```
line.replace(
                    f'TO {from_config["user"]}',
                    f'TO {to_config["user"]}',
                ),
```

Reply:
And I couldn't think of any regex that would do `line.replace(from_schema, to_schema)` because schema names are all over the place in the dump file

Reply:
:+1: that's cool